config/application.rb

```diff
@@ -52,7 +52,7 @@ class Application < Rails::Application
     config.action_view.default_form_builder = GOVUKDesignSystemFormBuilder::FormBuilder
 
     config.i18n.exception_handler = Proc.new { |exception| raise exception.to_exception }
-    config.action_view.raise_on_missing_translations = true
+    config.i18n.raise_on_missing_translations = true
```

Comment:
Does this make https://github.com/DFE-Digital/apply-for-teacher-training/pull/4220 redundant?

Reply:
Unfortunately still need the custom exception handler to also raise errors in the case of activemodel errors that don't resolve correctly and don't have a default (eg — errors for custom validations).